scalableminds/webknossos
# Avoid more null-errors in isosurface and mesh handling

The meshes UI could provoke errors when viewing a dataset without segmentation layer or when update-permissions are missing (for example, in view mode).

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- I tested the cross-product of the following factors:
  - dataset with and without segmentation layer (I used ROI2017_wkw and data_types)
  - open dataset in view mode, with a hybrid and with a skeleton-only annotation
- In each constellation:
  - make sure the "load centered isosurface" behaves correctly (it should only be disabled if there is no segmentation layer). if the centered cell is 0, the button is available, but should emit a warning toast.
  - the import buttons are only enabled if allowUpdate is true
  - importing an meshes/isosurface STLs should "work as expected". see [here](https://www.notion.so/scalableminds/Test-Data-c0563be9c4a4499dae4e16d9b2497cfb#f3ca867baabf43cca7f562ec16039a5a) for sample STLs.

### Issues:
- fixes #5106
- fixes #4935

------
